backend/queries/cohorts.js

```diff
@@ -19,6 +19,15 @@ const selectAllCohorts = async () => {
   return await db.any(getQuery);
 }
 
+const selectCohortById = async (id) => {
+  const getQuery = `
+    SELECT *
+    FROM cohorts
+    WHERE cohort_id = $/id/;
+  `;
```

Comment:
soft delete condition missing?

Reply:
Yes, removed it, we still want to display a cohort if searched by Id even if it was deleted for relation fellow-cohort to display fellows profiles